### PR TITLE
Add IPv6 basic config and bridge tests 

### PR DESCRIPTION
**Pull request by SerhiyBoikoPLV**
- test_ipv6_basic_config
- test_ipv6_move_host_on_bridge
- test_ipv6_on_bridge
- test_ipv6_on_bridge_vlan

Signed-off-by: Serhiy Boiko <serhiy.boiko@plvision.eu>

### CODE REVIEW

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+    dut_routes = [info.swp_ip[:-1] + f"/{info.plen}" for info in address_map]
+    for route in out[0][dent]["parsed_output"]:
+        if route["dev"] not in ports:
+            continue
+        if route["dst"].startswith("f"):  # broadcast
+            continue
+        assert "rt_trap" in route["flags"], "Route should be offloaded"
+        assert route["dst"] in dut_routes, \
+            f"Expected route to be one of {dut_routes}, not {route['dst']}"
```
Comment (yuriilisovskyi) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py]: I assume this piece of code is used in several places, move it to ipv6_utils.py

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+    dut_nei = [info.tg_ip for info in address_map]
+    for nei in out[0][dent]["parsed_output"]:
+        if nei["dev"] not in ports:
+            continue
+        if nei["dst"].startswith("f"):  # broadcast
+            continue
+        assert nei["dst"] in dut_nei, f"Expected neighbor to be one of {dut_nei}"
+        assert "offload" in nei, "Neighbor should be offloaded"
+        assert nei["state"] != "FAILED", "Unexpected neighbor state"
```
Comment (yuriilisovskyi) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py]: If it is used in several places, move it to utils

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+
+
+async def test_ipv6_on_bridge_vlan(testbed):
```
Comment (yuriilisovskyi) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py:154]: Let's keep the rule "one file - one test" :)

**Review by yuriilisovskyi — CHANGES_REQUESTED** [review]

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py (hunk 1 above)
Comment (SerhiyBoikoPLV) [reply DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py]: moved to [ipv6_utils.py](https://github.com/PLVision/dent-testing/pull/86/files#diff-c406f4edaecdfea65f8c5ad01d9cf080349d2f0b6ef4662351c96105e6b2ad0b)

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py (hunk 2 above)
Comment (SerhiyBoikoPLV) [reply DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py]: moved to [ipv6_utils.py](https://github.com/PLVision/dent-testing/pull/86/files#diff-c406f4edaecdfea65f8c5ad01d9cf080349d2f0b6ef4662351c96105e6b2ad0b)

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py (hunk 3 above)
Comment (SerhiyBoikoPLV) [reply DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv6/test_ipv6_bridge.py:154]: as discussed, similar tests will be kept in the same file

**Review by yuriilisovskyi — APPROVED** [review]

**Pull request closed without merging**